app/__init__.py

```diff
@@ -19,13 +20,14 @@ def create_app(config_name):
                         static_folder='static/',
                         static_url_path=configs[config_name].STATIC_URL_PATH)
 
-    application.config.from_object(configs[config_name])
-    configs[config_name].init_app(application)
-    config.init_app(application)
-
-    bootstrap.init_app(application)
-    logging.init_app(application)
-    data_api_client.init_app(application)
+    init_app(
+        application,
+        configs[config_name],
+        bootstrap=bootstrap,
+        data_api_client=data_api_client,
+        feature_flags=feature_flags
+    )
+    # Should be incorporated into digitalmarketplace-utils as well
     csrf.init_app(application)
```

Comment:
I'm not sure we have to worry too much about removing duplication here. Not all applications need `csrf` and all the `csrf` setup is done in `csrf.init_app` so making each app that does require it call `csrf.init_app` seems better than passing it into another function.